OsLogin/synth.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+common = gcp.CommonTemplates()
+
+v1beta = gapic.php_library(
+    service='oslogin',
+    version='v1beta',
+    artman_output_name='google-cloud-os-login-v1beta')
+
+v1 = gapic.php_library(
+    service='oslogin',
+    version='v1',
+    artman_output_name='google-cloud-os-login-v1')
+
+# copy all src
+s.move(v1beta / f'src/V1beta')
+s.move(v1 / f'src/V1')
+
+# copy proto files to src also
+s.move(v1beta / f'proto/src/Google/Cloud/OsLogin', f'src/')
+s.move(v1 / f'proto/src/Google/Cloud/OsLogin', f'src/')
+
+s.move(v1beta / f'tests/')
+s.move(v1 / f'tests/')
+
+# copy GPBMetadata file to metadata
+s.move(v1beta / f'proto/src/GPBMetadata/Google/Cloud/Oslogin', f'metadata/')
+s.move(v1 / f'proto/src/GPBMetadata/Google/Cloud/Oslogin', f'metadata/')
+
+# Remove @experimental tags from the V1 code
```

Comment:
As noted above, based on our discussions with @michaelbausor the plan was just for VI to go GA at this time - so I believe we want to keep these tags.

Reply:
Alright, let's keep this beta